doc/modules/cross_validation.rst

```diff
@@ -240,16 +240,18 @@ prediction that was obtained for that element when it was in the test set. Only
 cross-validation strategies that assign all elements to a test set exactly once
 can be used (otherwise, an exception is raised).
 
-These prediction can then be used to evaluate the classifier::
 
-  >>> from sklearn.model_selection import cross_val_predict
-  >>> predicted = cross_val_predict(clf, iris.data, iris.target, cv=10)
-  >>> metrics.accuracy_score(iris.target, predicted) # doctest: +ELLIPSIS
-  0.973...
+Note on inappropriate usage of cross_val_predict
+================================================
+The function :func:`cross_val_score` takes an average of over cross-validation
+folds and is grouped in different ways on computation in comparison to :func:`cross_val_predict`.
+Any metric score on :func:`cross_val_predict` is not equivalent to the
+result of :func:`cross_val_score`. Thus, the function :func:`cross_val_predict` is
+not an appropriate measure of generalisation error.
+
+The valid use of :func:`cross_val_predict` is for visualization of
+training set predictions and model blending techniques.
```

Comment:
I think this is again vague. Try to be more specific. Look at where `cross_val_predict` is currently used in the library, and try to explain it in your own words.

Reply:
Will do.